dotCMS/src/main/java/com/dotmarketing/db/CommitListenerThread.java

```diff
@@ -0,0 +1,63 @@
+package com.dotmarketing.db;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import com.dotmarketing.business.DotStateException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.util.Config;
+
+public class  DotRunnableThread extends Thread {    
+    
+    final List<DotRunnable> listeners;
+    public DotRunnableThread(List<DotRunnable> listeners){
+        
+        this.listeners=listeners;
+    }
+      @Override
+      public void run() {
```

Comment:
I think it can be changed to 
```
@WrapInTransaction
public void run() {
     _internalRunner();
}
```

Reply:
yes for master, don't have that annotation in project-8

Reply:
ohhh I see :+1: